src/index.css

```diff
@@ -1,11 +1,42 @@
 
-form {
-   border-radius: 13px;
-   border: solid;
-   border-width: 1px;
-   padding: 50px;
-}
- 
-h1{
-    text-align: center;
-}
\ No newline at end of file
+
+    body {
+      background-color: #fcfbf8;
+      background-image: url('./images/peakpx.jpg');
+      background-repeat: no-repeat; 
+      background-size: cover; 
+      background-position: center
+    }
+
+    .signup-container, .signin-container {
+      max-width: 400px;
+      margin: auto;
+      padding: 20px;
+      border: 1px solid #f37805;
+      border-radius: 30px;
+      background-color: rgba(103, 107, 103, 0.568);
+
+    }
+
+    .logo{
+      width: 70%;
+      display: block;
+      margin: auto;
+    }
+   
+
+    h2{
+      text-align: center;
+    }
+
+    
+
+    #account-button{
+      margin-left: 15px;
+      background-color:rgba(253, 78, 9, 0.753);
+      border-radius: 10px;
+      height: 40px;
+      border: 0px;
+      font-style: italic;
+    }
+ 
```

Comment:
Avoid using global styles to style specific components, as this may lead to conflicts and limit your flexibility. It's better to create small styles.css file for each component and only add styles specific to it. Then the global styles will only be for things you want to apply to all pages of your app.

Reply:
It's better to create a folder for each component under components. For example a folder called signin can contain the Signin component and a styles.css file that you only import in the Signin component. Same for other components. This would make it easier for you to know which styles you've applied on which components.
@MbiydzenyuyStephen , @sakahmercy : pls take note of this comment too